### DISCUSSION TITLE
post/1/

### DISCUSSION THREAD

**Discussion by giscus** [HoshinoRei/blog#15] [General]
# post/1/

Windows程序包管理器（英语：Windows Package Manager，也称 winget）是微软为 Windows 10 开发的一款自由开源的软件包管理器。它由一个命

https://blog.bling.moe/post/1/

**Comment by wcmiracle** [comment]
使用 winget 批量安装软件包  这里实测没有用。我加了分号。C:\Users\a>winget install Google.Chrome;
找不到与输入条件匹配的程序包。

C:\Users\a>winget install Obsidian.Obsidian

**Reply by wcmiracle** [reply]
第一条安装指令他把分号也当成包名了

**Reply by HoshinoRei** [reply]
试试 `--id`。
